contrib/storage-druid/src/main/java/org/apache/drill/exec/store/druid/common/DruidBoundFilter.java

```diff
@@ -0,0 +1,123 @@
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.store.druid.common;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.HashMap;
+
+public class DruidBoundFilter {
+
+  private static final ObjectMapper objectMapper = new ObjectMapper();
+  private String type = DruidCompareOp.TYPE_BOUND.getCompareOp();
+  private String dimension;
+  private String lower;
+  private String upper;
+  private Boolean lowerStrict =  false;
+  private Boolean upperStrict = false;
+
+  @JsonCreator
+  public DruidBoundFilter(@JsonProperty("dimension") String dimension,
+                          @JsonProperty("lower") String lower,
+                          @JsonProperty("upper") String upper) {
+    this.dimension = dimension;
+    this.lower = lower;
+    this.upper= upper;
+  }
+
+  @JsonCreator
+  public DruidBoundFilter(@JsonProperty("dimension") String dimension,
+                          @JsonProperty("lower") String lower,
+                          @JsonProperty("upper") String upper,
+                          @JsonProperty("lowerStrict") Boolean lowerStrict,
+                          @JsonProperty("upperStrict") Boolean upperStrict) {
+    this.dimension = dimension;
+    this.lower = lower;
+    this.upper= upper;
+    this.lowerStrict = lowerStrict;
+    this.upperStrict = upperStrict;
+  }
+
+  public String getType() {
+    return type;
+  }
+
+  public void setType(String type) {
+    this.type = type;
+  }
+
+  public String getDimension() {
+    return dimension;
+  }
+
+  public void setDimension(String dimension) {
+    this.dimension = dimension;
+  }
+
+  public String getLower() {
+    return lower;
+  }
+
+  public void setLower(String lower) {
+    this.lower = lower;
+  }
+
+  public String getUpper() {
+    return upper;
+  }
+
+  public void setUpper(String upper) {
+    this.upper = upper;
+  }
+
+  public Boolean getLowerStrict() {
+    return lowerStrict;
+  }
+
+  public void setLowerStrict(Boolean lowerStrict) {
+    this.lowerStrict = lowerStrict;
+  }
+
+  public Boolean getUpperStrict() {
+    return upperStrict;
+  }
+
+  public void setUpperStrict(Boolean upperStrict) {
+    this.upperStrict = upperStrict;
+  }
+
+  public String toJson() throws JsonProcessingException {
```

Comment:
Interesting. This class is configured for Jackson, but the JSON is created by hand in this method. Any reason to not use Jackson to do the work?

Reply:
Fixed, I think i was eating bourbon burgers at times.